notebooks/2_4_highlight_fragments_in_ligand.ipynb

```diff
@@ -8,8 +8,6 @@
     "\n",
```

Comment:
<p>The link to Greg's blog post is not working anymore, can you check if you can find the blog post and the link just changed? If it's taken down, we can just rephrase this.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/KinFragLib/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>No, I've found the new link</p>